courses/models_test.py

```diff
@@ -286,3 +288,18 @@ def test_course_page_properties():
         assert block.value == "benefit"
     assert course.outcomes == learning_outcomes_page
     assert not LearningOutcomesPage.can_create_at(course_page)
+
+
+def test_course_page_faq_property():
```

Comment:
please add test for the corresponding program properties too.

Reply:
added